devel/cocogitto/Portfile

```diff
@@ -0,0 +1,288 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        cocogitto cocogitto 6.3.0
+github.tarball_from archive
+revision            0
+
+categories          devel
+license             MIT, Apache-2
+maintainers         {macports.halostatue.ca:austin @halostatue} \
+                    openmaintainer
+
+description         The Conventional Commits toolbox
+long_description    {*}{
+  The Conventional Commits toolbox, providing verified commits, automatic version bump and
+  changelog, release profiles, conventional git log, GitHub integration, and monorepo
+  support.
+}
+
+checksums           ${distname}${extract.suffix}  \
+                    rmd160  542ccdaa3e27c96aabfebabdc1856f331ba64a03 \
+                    sha256  bf78a06ec20cd33c4f9bcddb427067de34d005fb6d4a41727239b7b1e8e916e0 \
+                    size    1771859
+
+installs_libs       no
+use_configure       no
```

Comment:
Super minor nitpick, but since you're using the `cargo` portgroup, `use_configure` will be `no` by default.